packages/forms-web-app/__tests__/unit/controllers/projects/examination-timetable/utils/events/event/utils/get-event-state.test.js

```diff
@@ -0,0 +1,62 @@
+const {
+	getEventState
+} = require('../../../../../../../../../src/controllers/projects/examination-timetable/utils/events/event/utils/get-event-state');
+
+const {
+	isEventDeadlineSubmissionOpen,
+	isPastEventTypeDeadline
+} = require('../../../../../../../../../src/controllers/projects/examination-timetable/utils/events/event/helpers');
+
+jest.mock(
+	'../../../../../../../../../src/controllers/projects/examination-timetable/utils/events/event/helpers',
+	() => ({
+		isEventDeadlineSubmissionOpen: jest.fn(),
+		isPastEventTypeDeadline: jest.fn()
+	})
+);
+
+describe('controllers/projects/examination-timetable/utils/events/event/utils/get-event-state', () => {
+	describe('#getEventState', () => {
+		describe('When getting the event state', () => {
+			const mockEvent = {};
+			describe('and the event is not a deadline or the event deadline has not started', () => {
+				let result;
+				beforeEach(() => {
+					isEventDeadlineSubmissionOpen.mockReturnValue(false);
+					isPastEventTypeDeadline.mockReturnValue(false);
+					result = getEventState(mockEvent);
+				});
+				it('should return null', () => {
+					expect(result).toEqual({ isSubmissionOpen: false, tag: null });
```

Comment:
maybe could read - 'should submission open = false and a tag of null'

or something like that.